packages/@tinacms/scripts/src/index.ts

```diff
@@ -30,7 +30,12 @@ interface Option {
 
 const deepMerge = (target, source) => {
   for (const key in source) {
-    if (!source.hasOwnProperty(key) || key === '__proto__' || key === 'constructor') continue;
+    if (
+      !source.hasOwnProperty(key) ||
+      key === '__proto__' ||
+      key === 'constructor'
+    )
+      continue
```

Comment:
**issue**: If `continue` is to broken out to its own line, please use braces as per https://www.ssw.com.au/rules/how-to-avoid-problems-in-if-statements/

Reply:
better to have an eslint rules turned on for this
https://eslint.org/docs/latest/rules/brace-style
https://eslint.org/docs/latest/rules/curly

Reply:
I have reverted the change for now as its not relevant to this PR